modules/installation-aws-ami-stream-metadata.adoc

```diff
@@ -0,0 +1,31 @@
+//TODO: Add the module include to the following assemblies
+//TODO: Create related modules for OpenStack (QCOW2 and Bare Metal ISO)
+
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+
+[id="installation-aws-ami-stream-metadata_{context}"]
+= Printing {op-system} AMIs for AWS zones by using stream metadata
+
+In {product-title}, _stream metadata_ provides standardized metadata about {op-system} in the JSON format and injects the metadata into the cluster. Stream metadata is a stable format that supports multiple architectures and is intended to be self-documenting for maintaining automation.
+
+Stream metadata is available in a Go library that contains example code for the software. The official `stream-metadata-go` library and example code is available at https://github.com/coreos/stream-metadata-go.
+
+You can use the `coreos print-stream-json` sub-command of `openshift-install` to access information about the bootimages in the stream metadata format. This command provides a method for printing stream metadata in a scriptable, machine-readable format. For user-provisioned installations, the `openshift-install` binary contains pinned versions of {op-system} bootimages, such as an AWS AMI.
+
+.Procedure
+
+* Print the current `x86_64` AMI for an AWS zone, such as `us-west-1`:
++
+[source,terminal]
+----
+$ openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.images.aws.regions["us-west-1"].image'
+----
++
+.Example output
+[source,terminal]
+----
+ami-0b421328859954025
```

Comment:
Gathered from lines [99-100](https://github.com/openshift/installer/commit/b64d699a13669e36e85e4cfe6fa6313202a7a690#diff-8749db54915c027b82a257627c6183c8edeb0c3b24a4b4a21a0bce8e53f180e9). I couldn't test this step, so please verify that this is the expected command syntax and output. 

Also, are there any other explicit steps that one should take from here? Or should we say anything about how to use the AMI output in scripting for automation?

Reply:
Whoops, I might have been on 4.7 binary. It worked of course with 4.8.

I haven't been involved in the manual update to the AMI table before (e.g., https://github.com/openshift/openshift-docs/pull/29337) but this is really cool. I ran it and could get all 4.8 AMIs:
```
$ stream openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.images.aws.regions[].image'           
ami-08c63bb1b8b661ab9
ami-01f2a0e8aee56e1f4
ami-012420354c0b2bf59
ami-0d8c0ce731da9faaf
ami-00cac27e65ae02107
ami-05b69d7e80f1178c6
ami-0bd576d6deb6094c8
ami-06e7e674d4b8cdc59
ami-04990181ab3872dc9
ami-0ed56edc094183f18
ami-0e63e7e6de6dac812
ami-01c63dbdcd28938c0
ami-04885ff040bf8e8d0
ami-04b31bbfe532cbb45
ami-0e188558a41f87d82
ami-006b4ea9d3e62fdc2
ami-094888ec0f3125ad7
ami-0b08ba24af2f005c9
ami-0cd5c6a0f8bb3c33d
ami-0d3e625f84626bbda
ami-040e8d23c125e7a75
```
Cc: @codyhoag @kalexand-rh

Reply:
Or just the `us-west-1` zone as described in the procedure:
`$ openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.images.aws.regions["us-west-1"].image'`
`ami-0d3e625f84626bbda`